src/main/java/frc/robot/AutonomousPath.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot;
+
+public class AutonomousPath{
+
```

Comment:
you didnt make a constructor that accepts a visionpathname ( making it required) like we discussed